aws-opentelemetry-distro/src/amazon/opentelemetry/distro/exporter/aws/metrics/base_emf_exporter.py

```diff
@@ -184,6 +191,28 @@ def _get_dimension_names(self, attributes: Attributes) -> List[str]:
         # For now, use all attributes as dimensions
         return list(attributes.keys())
 
+    def _add_lambda_dimensions(self, emf_log: Dict, dimension_names: List[str], resource: Resource) -> List[str]:
+        """
+        Add Service and Environment dimensions for Lambda environments.
+
+        Args:
+            emf_log: The EMF log dictionary to add dimension values to
+            dimension_names: The current list of dimension names
+            resource: The OTel Resource to extract service name from
+
+        Returns:
+            Updated list of dimension names with Service and Environment prepended (if Lambda)
+        """
+        if not self._is_lambda:
+            return dimension_names
+
+        service_name, _ = get_service_attribute(resource) if resource else ("UnknownService", True)
+        # Add dimension values to the EMF log root
+        emf_log["Service"] = service_name
+        emf_log["Environment"] = LAMBDA_ENVIRONMENT_DEFAULT
```

Comment:
Environment can be customized by setting resource attributes. Do we have a similar function as `get_service_attribute` for environment?